cdk/lib/prism-access.ts

```diff
@@ -29,6 +29,7 @@ export class PrismAccess extends GuStack {
     };
 
     const prismRole = new GuRole(this, "PrismRole", {
+      existingLogicalId: "PrismRole",
```

Comment:
nitpick: worth adding a comment here to explain why the default id is undesired?

This prop should really be changed to enforce this:

```typescript
interface ExistingLogical {
  value: string;
  description: string;
}
```